src/Domains/index.js

```diff
@@ -21,10 +21,22 @@ export default class Domains {
     for(appId) {
         return unpackData(this._client.apiClient().get(`/apps/${appId}/domains`), "domains")
     }
+
+    /**
+     * Link a domain name to a specific application
+     * @see http://developers.scalingo.com/domains#link-a-domain-name-to-an-application
+     * @param {String} appId ID of the app to post to the specified application
+     * @param {Domain[]} domain An object of the domain to link
+     * @return {Promise<Domain[] | APIError>}
+     */
+    create(appId, domain) {
+        return unpackData(this._client.apiClient().post(`/apps/${appId}/domains`, {domains: domain}), "domains")
```

Comment:
```suggestion
        return unpackData(this._client.apiClient().post(`/apps/${appId}/domains`, {domain: domain}), "domain")
```

(see http://developers.scalingo.com/domains#link-a-domain-name-to-an-application)